src/routes/error.js

```diff
@@ -0,0 +1,12 @@
+module.exports = [{
+  method: ['GET', 'POST'],
```

Comment:
At the time of writing, since the error page was only a placeholder and not designed yet I thought there might be a possibility of needing a POST, but on reflection I agree that this is unlikely so I have taken it out and made it GET only.